cmd/relayproxy/api/middleware/body.go

```diff
@@ -0,0 +1,19 @@
+package middleware
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/thomaspoignant/go-feature-flag/cmd/relayproxy/config"
+	"go.uber.org/zap"
+)
+
+func BodyLogger(log *zap.Logger, cfg *config.Config) echo.MiddlewareFunc {
+	skipper := func(_ echo.Context) bool { return !cfg.IsDebugEnabled() }
+
+	return middleware.BodyDumpWithConfig(middleware.BodyDumpConfig{
+		Skipper: skipper,
+		Handler: func(_ echo.Context, req []byte, _ []byte) {
+			log.Debug("Request", zap.ByteString("body", req))
```

Comment:
I am not a fan of using `body` as the property name here I would go more with `request_body`, so we can evolve this log in the future.

```suggestion
			log.Debug("Request info", zap.ByteString("request_body", reqBody))
```